docs/sources/api-response-examples/microsoft-365/directory/sanitized-no-app-ids/user.json

```diff
@@ -6,12 +6,12 @@
   "officeLocation":"12/1110",
   "preferredLanguage":"en-US",
   "userPrincipalName":"{\"scope\":\"email\",\"domain\":\"M365x214355.onmicrosoft.com\",\"hash\":\".tbhph3DloGSd5l9e9TeFqmEboAqU8o6BdigmHZOmxI\"}",
-  "id":"p~SIoJOpeSgYF7YUPQ28IWZexVuHyN9A80SJXrbfawKpDRcddGnKI4QDKyjQI9KtjJZDb8FZ27UE_toS68FyWz7Y22fnQYLP91SHJGVwQiN3E",
+  "id":"{\"scope\":\"azure-ad\",\"hash\":\"J0_C7jfy9KhwcfmhAjVNQOnzUnYaE3Kovzt2FAkc2W0\",\"reversible\":\"p~SIoJOpeSgYF7YUPQ28IWZexVuHyN9A80SJXrbfawKpDRcddGnKI4QDKyjQI9KtjJZDb8FZ27UE_toS68FyWz7Y22fnQYLP91SHJGVwQiN3E\"}",
```

Comment:
so if anyone were to use the "no app id" rules and 'LEGACY' format, the `hash` parsed from `reversible` won't match the non-reversible cases.

change in test here reflects that for LEGACY case, we're fail back to JSON. (tests, regretably, mostly run under `LEGACY` implementation)

Reply:
(for reference, `LEGACY` implementation uses hash of `(scope + secret + value)`; default is `(secret + value)`); `reversible` is computed at layer of code that is unaware of `scope`, and would be big refactor to put it there to account for very small number of people using `LEGACY` pseudonyms, which was v0.3)